tests/test_flow/fakes.py

```diff
@@ -166,8 +169,40 @@ def rm(self, url, recursive=False):
             del self._files_by_url[url]
 
 
+def move_files_in_dir(src_dir, dst_dir):
```

Comment:
I think it is doing the same thing as `bionic.utils.files.recursively_copy_path`. Can you use the util method if it works in place of this one?

Reply:
There is a difference. Firstly it is a move operation. Secondly, we want to move the contents of the source directory instead of moving the directory itself. For example if we have a file `src/a/b`, we want the resulting path to be `dest/a/b` instead of `dest/src/a/b`.